bootstrap/namespaces/helmfile/values/namespaces-sc.yaml.gotmpl

```diff
@@ -2,6 +2,7 @@ namespaces:
 - name: cert-manager
   labels:
       certmanager.k8s.io/disable-validation: "true"
```

Comment:
Do we still need this if it's changed to the new one?

Reply:
I guess not, but I prefer changing it "slowly". We can remove it later once we have a stronger QA in place.